submissions/2024-12-12-OpenFF-Sage-2.0.0-Training-Optimization-Dataset-v1.0/generate-combined-dataset.py

```diff
@@ -12,156 +14,179 @@
 from openff.toolkit.utils import OpenEyeToolkitWrapper, ToolkitRegistry
 from openff.units import unit
 
-def pull_record_id_cmiles(Opt):
 
+def pull_record_id_cmiles(Opt: Type[OptimizationResultCollection]):
```

Comment:
(non-blocking since the type signature won't affect performance) -- I hadn't actually seen typing.Type used before so had to look it up. It looks like it's an alias of `type`? If so I think `Opt: OptimizationResultCollection` -- this tells the typer that `Opt` should be an object of OptimizationResultCollection, whereas `type` would refer to a class itself.

Reply:
If black did this automatically, it might be because of the capitalization of Opt confusing it.